product-service-ts/swagger.config.yml

```diff
@@ -0,0 +1,80 @@
+swagger: '2.0'
+info:
+  description: This is a simple API
+  version: 1.0.1
+  title: Simple Inventory API
+  # put the contact info for your development or API team
+  contact:
+    email: you@your-company.com
+
+  license:
+    name: Apache 2.0
+    url: http://www.apache.org/licenses/LICENSE-2.0.html
+
+# tags are used for organizing operations
+paths:
+  /products/{id}:
+    get:
+      tags:
+      - getProductsById
+      summary: Get specific product by id (title)
+      operationId: searchProduct
+      description: |
+        By passing in product title, you can search for
+        available product in the system
+      produces:
+      - application/json
+      parameters:
+      - in: path
+        name: id
+        description: pass an product title in order to find specific product
+        required: true
+        type: string
+      responses:
+        200:
+          description: OK
+          schema:
+            $ref: '#/definitions/GetProductsById'
+        404:
+          description: No product with such id
+  /products/:
+    get:
+      tags:
+      - getProductsList
+      summary: Get all products
+      operationId: addProducts
+      description: Get all products to the system
+      produces:
+      - application/json
+      responses:
+        200:
+          description: OK
+          schema:
+            items:
+              $ref: '#/definitions/GetProductsById'
+definitions:
+  GetProductsById:
```

Comment:
In definition should be Model (product) not a method (GetProductsById)

Reply:
Thanks, done.